### PR TITLE
 🌐 Add Portuguese translation for `docs/pt/docs/tutorial/body-updates.md`

### PR DESCRIPTION
**Pull request by andersonrocha0**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

File: docs/pt/docs/tutorial/body-updates.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+///
+
+### Usando o parâmetro `exclude_unset` do Pydantic
+
+Se você quiser receber atualizações parciais, é muito útil usar o parâmetro `exclude_unset` no método `.model_dump()` do modelo do Pydantic.
+
+Como `item.model_dump(exclude_unset=True)`.
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/body-updates.md:75]: It's good to add the info translation too:

```markdown
/// info | Informação
```

File: docs/pt/docs/tutorial/body-updates.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+{!> ../../../docs_src/body_updates/tutorial002.py!}
+```
+
+////
+
+### Usando o parâmetro `update` do Pydantic
+
+Agora, você pode criar uma cópia do modelo existente usando `.model_copy()`, e passar o parâmetro `update` com um `dict` contendo os dados para atualizar.
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/tutorial/body-updates.md:115]: the same here: /// info | Informação

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great 🚀

File: docs/pt/docs/tutorial/body-updates.md (hunk 1 above)
Comment (andersonrocha0) [reply docs/pt/docs/tutorial/body-updates.md]: Hello, thanks for the advice. Could you please check it now?

File: docs/pt/docs/tutorial/body-updates.md (hunk 2 above)
Comment (andersonrocha0) [reply docs/pt/docs/tutorial/body-updates.md]: Same here.

File: docs/pt/docs/tutorial/body-updates.md (hunk 1 above)
Comment (andersonrocha0) [reply docs/pt/docs/tutorial/body-updates.md]: @ceb10n I have already applied your suggestions. Could you please check it again?

**Review by ceb10n — APPROVED** [review]
🚀

**Pull request merged**